lib/auto-ssl.js

```diff
@@ -0,0 +1,44 @@
+import fs from "fs"
+import { spawnSync } from "child_process"
+
+export default function (host, email, linkDest) {
+  //where simp_le should create any keys, certs, etc
+  const sourceDest = `/etc/ssl/letsencrypt/${host}`;
+
+  //create sourceDest as necessary
+  try {
+    fs.accessSync(sourceDest);
+  } catch(e) {
+    fs.mkdirSync(sourceDest);
+  }
+
+  //try to generate cert via simp_le
+  let proc = null;
+  try {
+    proc = spawnSync("/var/simp_le/venv/bin/simp_le", [
```

Comment:
am I right in assuming that this will check for the cert existing before creating it?  I wonder if that check might be better done in node